doc/holochain_101/src/dpki.md

```diff
@@ -0,0 +1,54 @@
+# Distributed Public Key Infrastructure (DPKI)
+
+## Context
+
+Like most other distributed systems, Holochain fundamentally relies on public key cryptography. Among other uses, Holochain nodes are identified by their public keys, and thus provenance of messages from nodes can be checked simply by checking a message signature against the node's identifier.  This fundamental use requires nodes to keep the private key secret, lest the node's very agency be compromised.  Keeping such secrets in the digital age is a non-trivial problem.  Additionally, distributed systems create a context of a profusion of nodes with multiple public keys, many of which may wish to be identified as being under the control a of a single actor.  Solutions to this need create yet another layer of keys which sign other keys, and the management of all this can become quite complex.
```

Comment:
```suggestion
Like most other distributed systems, Holochain fundamentally relies on public key cryptography. Among other uses, Holochain nodes are identified by their public keys, and thus provenance of messages from nodes can be checked simply by checking a message signature against the node's identifier.  This fundamental use requires nodes to keep the private key secret, lest the node's very agency be compromised.  Keeping such secrets in the digital age is a non-trivial problem.  Additionally, distributed systems imply an abundance of nodes with public keys, many of which may wish to be identified as being under the control of a single actor.  Solutions to this need create yet another layer of keys which sign other keys, and the management of all this can become quite complex.
```